fastlane/Fastfile

```diff
@@ -10,6 +10,8 @@ end
 USER_ENV_FILE_PATH = File.join(Dir.home, '.simplenoteios-env.default')
 PROJECT_ENV_FILE_PATH = File.expand_path(File.join(Dir.pwd, '../.configure-files/project.env'))
 UI_TESTS_SCHEME = "SimplenoteUITests"
+UI_TESTS_ACCOUNTS_KEY = "https://api.keyvalue.xyz/82649691/simplenoteuitest-ios"
```

Comment:
Nitpick. What do you think of using `URL` in the name? The value is a URL more than a key.

```suggestion
UI_TESTS_ACCOUNTS_URL = "https://api.keyvalue.xyz/82649691/simplenoteuitest-ios"
```

Reply:
Done! Make sense, thank you